tests/tests_lite/helpers/runif.py

```diff
@@ -12,12 +12,22 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 import os
+import sys
+from typing import Optional
 
 import pytest
 import torch
+from packaging.version import Version
+from pkg_resources import get_distribution
+
+from lightning_lite.utilities.imports import (
+    _FAIRSCALE_AVAILABLE,
+    _PSUTIL_AVAILABLE,
+    _TORCH_GREATER_EQUAL_1_10,
+    _TPU_AVAILABLE,
+)
 
 
-# TODO(lite): Add all RunIf conditions once the relevant utilities have moved to lite source dir
```

Comment:
@akihironitta move this to testing utilities?

Reply:
The RunIf for each library will likely have different checks and dependencies. We have seen this in the past, e.g. with the RunIf version of the lightning_app framework, where it has radically different implementation.